src/EventPayload.ts

```diff
@@ -118,7 +147,11 @@ export interface EventPayload {
     /* The ID of the template from which a site was generated. */
     template?: string;
   };
-  /** The timestamp at which the event occurred, in ISO format. */
+  /** The timestamp at which the event occurred, in ISO format.
```

Comment:
Might be useful to provide a template or example

Reply:
Discussed here https://yext.slack.com/archives/G01SSAYJVM4/p1709063238616539